pkg/config/types/requester.go

```diff
@@ -53,3 +55,20 @@ type S3StorageProviderConfig struct {
 type JobDefaults struct {
 	ExecutionTimeout Duration `yaml:"ExecutionTimeout"`
 }
+
+type RequesterControlPlaneConfig struct {
```

Comment:
Again, naming is hard. Maybe `HeartbeatServerConfig`? based on the constructor that accepts this?